Intelipro/reja
# Feat: Display segment values

#### What does this PR do?
 Displays all the values available on the customer segment

#### Description of Task to be completed?
- [x]  Add cards to display more values

#### How should this be manually tested?

- Clone this repo git clone [the-repo-link]
- Checkout to this branch `ft-segment-values`
- Install all dependencies npm install
- Start the server npm start
- To access the page go to http://localhost:3000 on your browser
- Log in to access the application
- Navigate to the customer page under the analytics sidebar item. Below the graph, you'll see the segment section
- Click the various value level(low, mid, high) to see the added values with different colors

#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)

#### Questions:
